base/applications/cleanmgr/cleanmgr/cleanmgr.rc

```diff
@@ -0,0 +1,65 @@
+/*
+ * PROJECT:     ReactOS Disk Cleanup
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Resources
+ * COPYRIGHT:   Copyright 2023 Mark Jansen <mark.jansen@reactos.org>
+ */
+
+#include <windef.h>
+#include <winuser.h>
+#include <commctrl.h>
+#include "resource.h"
+
+#define REACTOS_STR_FILE_DESCRIPTION  "ReactOS Disk Cleanup"
+#define REACTOS_STR_INTERNAL_NAME     "cleanmgr"
+#define REACTOS_STR_ORIGINAL_FILENAME "cleanmgr.exe"
+#include <reactos/version.rc>
+
+#include <reactos/manifest_exe.rc>
+
+IDI_CLEANMGR ICON "resources/cleanmgr.ico"
+
+#pragma code_page(65001)
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```

Comment:
What about putting that stuff in a separate en_US.rc file? Or do you want to stabilize the current resources before doing this?

Reply:
can be moved when translation starts, it's really annoying editing those files.